docs/source/javascripts/all.js

```diff
@@ -1,2 +1,17 @@
 places({container: document.querySelector('#landing-demo')});
 
+document.addEventListener('scroll', function(e){
+	var v = e.target.scrollingElement.scrollTop;
+	var h = document.querySelector('.hero-section').offsetHeight;
+	var nav = document.querySelector('.navigation').offsetHeight;
+	if(v>h) {
+		// document.querySelector('.navigation').classList.add('darken');
```

Comment:
I think using classes will make it clearer and less "magic hex code style in JS"